deploy/fabfile.py

```diff
@@ -33,26 +33,38 @@
 home = "/home/site"
 log = "%s/hookshot.log" % home
 current = "%s/%s/current" % (home, environment)
+ruby = "/opt/install/rbenv/shims/ruby"
 
-# principal command to run upon update
-command = "cd %s && git pull && bundle exec jekyll build >> %s" % (current, log)
+if environment == 'staging':
+  # on staging, build the site but also push back the extracted data files
+  command = "cd %s && git pull && %s _data/import-data && git add _data/team.yml _data/projects.yml && git commit -m 'update team and project data' && git push origin staging && %s go build >> %s" % (current, ruby, ruby, log)
```

Comment:
Why have the staging site do the data extraction and git pushing? It seems like deploys are more predictable (though less automatic) when doing this work locally. Even updating the fabric script to run the git download and add commands on the local machine (while still doing everything else remotely) would be more preferable, I think.